.dev/revdep-repos

```diff
@@ -0,0 +1,78 @@
+http://github.com/soodoku/abbyyR
+http://github.com/soodoku/tuber
```

Comment:
nit: https:// everywhere?

Reply:
this is taken directly from CRAN, but I have filed PRs to do this, so it'll be picked up eventually:

https://github.com/soodoku/abbyyR/pull/13
https://github.com/soodoku/tuber/pull/119